plugins/jenkins_status_checker/README.md

```diff
@@ -0,0 +1 @@
+This plugin allows you to see a list of failing jenkins jobs that may block a deploy prior to deploying. This only shows for production, same rules as the buddy check. It shares the jenkins credentials of the jenkins plugin.
```

Comment:
"Shows failing jenkins jobs when users try to create a new deploy."

Reply:
mention the settings it needs (including JENKINS_STATUS_CHECKER) and mention JENKINS_STATUS_CHECKER in .env.example